src/components/Type/Type.css

```diff
@@ -0,0 +1,119 @@
+.type-container-small {
+  display: flex;
+  width: 100%;
+}
+.type-container-large {
```

Comment:
this is the same as class above so we can probs delete this one

Reply:
fink we are gonna edit the ccs of these cants so they will be different, no?